components/page.js

```diff
@@ -68,54 +85,38 @@ export default ({ hackathons, currentPath, children }) => {
           {loading && <LoadingImage src={LoadingGif} />}
         </HeaderContainer>
         <HackathonNavBar>
-          <Link href="/[id]/intro" as={`/${currentPath}/intro`}>
+          <Link href="/[id]/intro" as={`/${currentPath}/intro`} passHref>
             <NavItem
-              onClick={() => {
-                if (!window.location.href.includes('intro')) {
-                  setTimeOut(
-                    setTimeout(() => {
-                      setLoading(true);
-                    }, 750)
-                  );
-                }
-              }}
+              onClick={() => onClick('intro')}
               selected={currPath === 'intro'}
             >
               Intro
             </NavItem>
           </Link>
-          <Link href="/[id]/faq" as={`/${currentPath}/faq`}>
+          <Link href="/[id]/faq" as={`/${currentPath}/faq`} passHref>
             <NavItem
-              onClick={() => {
-                if (!window.location.href.includes('faq')) {
-                  setTimeOut(
-                    setTimeout(() => {
-                      setLoading(true);
-                    }, 750)
-                  );
-                }
-              }}
+              onClick={() => onClick('faq')}
               selected={currPath === 'faq'}
             >
               FAQ
             </NavItem>
           </Link>
-          <Link href="/[id]/spocos" as={`/${currentPath}/spocos`}>
+          <Link href="/[id]/spocos" as={`/${currentPath}/spocos`} passHref>
             <NavItem
-              onClick={() => {
-                if (!window.location.href.includes('spocos')) {
-                  setTimeOut(
-                    setTimeout(() => {
-                      setLoading(true);
-                    }, 750)
-                  );
-                }
-              }}
+              onClick={() => onClick('spocos')}
               selected={currPath === 'spocos'}
             >
               Sponsors
             </NavItem>
           </Link>
+          <Link href="/[id]/FeatureFlags" as={`/${currentPath}/FeatureFlags`}>
```

Comment:
```suggestion
          <Link href="/[id]/FeatureFlags" as={`/${currentPath}/FeatureFlags`} passHref>
```

oops i forgot about this

Reply:
what does it do?

Reply:
Yea actually wondering this too 🤔

Reply:
it passes the href to the child element. If you preview this PR you'll notice that the feature flags link doesn't have an href tag and is therefore not keyboard targetable and the pointer doesn't show u can click on it
https://nextjs.org/docs/api-reference/next/link#if-the-child-is-a-custom-component-that-wraps-an-a-tag

Reply:
#36